### ISSUE TITLE
Highlight git commit merge conflict markers

### ISSUE THREAD

**Issue by salcode** [salcode/ironcode-vim#154]
```
match ErrorMsg '^\(<\|=\|>\)\{7\}\([^=].\+\)\?$'
```

from Vimways [Vim and Git](https://vimways.org/2018/vim-and-git/)

**Comment by salcode** [comment]
> Highlight merge conflict markers, and move through conflict markers with `]c` and `[c`

```
" Highlight merge conflict markers
match Todo '\v^(\<|\=|\>){7}([^=].+)?$'

" Jump to next/previous merge conflict marker
nnoremap <silent> ]c /\v^(\<\|\=\|\>){7}([^=].+)?$<CR>
nnoremap <silent> [c ?\v^(\<\|\=\|\>){7}([^=].+)\?$<CR>
```

From  [vimrcfu 177](https://vimrcfu.com/snippet/177)

**Comment by salcode** [comment]
When using the Syntax [Group Name](https://vimhelp.org/syntax.txt.html#%7Bgroup-name%7D) `Todo` I'm not seeing any highlighting with our colorscheme [sjl/badwolf](https://github.com/sjl/badwolf) (see #217)

![image](https://user-images.githubusercontent.com/5194588/121804542-e1860100-cc14-11eb-90bf-4daa0e1ae904.png)


Using `ErrorMsg` turns the text red

![image](https://user-images.githubusercontent.com/5194588/121804508-c1eed880-cc14-11eb-8706-bc732945b254.png)


Using `Error` uses reversed red text

![image](https://user-images.githubusercontent.com/5194588/121804530-d337e500-cc14-11eb-97a5-1be90193bc6c.png)

**Comment by salcode** [comment]
Spinning off 

> Highlight merge conflict markers, and move through conflict markers with `]c` and `[c`

into a separate issue #218 (and a separate PR)

**Comment by salcode** [comment]
Resolved in #219
